### ISSUE TITLE
When I try to import my model I run into an error "TypeError: PyMetaspace.__new__() got an unexpected keyword argument: str_rep"

### ISSUE THREAD

**Issue by IndianMLGay** [huggingface/transformers#10448]
In the Hugging Face repository I have my own model Fidlobabovic / beta-kvantorium-simple-small 

https://huggingface.co/Fidlobabovic/beta-kvantorium-simple-small/tree/main
#7370 
#10148 
When I try to import it, I get an error. What can I do to fix it? Overwrite model files or rename them? What should I write for this?
````
from transformers import AutoTokenizer, AutoModelForMaskedLM

tokenizer = AutoTokenizer.from_pretrained("Fidlobabovic/beta-kvantorium-simple-small")

model = AutoModelForMaskedLM.from_pretrained("Fidlobabovic/beta-kvantorium-simple-small")

TypeError                                 Traceback (most recent call last)
<ipython-input-5-3f4375d4cdf7> in <module>()
      1 from transformers import AutoTokenizer, AutoModelForMaskedLM
      2 
----> 3 tokenizer = AutoTokenizer.from_pretrained("Fidlobabovic/beta-kvantorium-simple-small")
      4 
      5 model = AutoModelForMaskedLM.from_pretrained("Fidlobabovic/beta-kvantorium-simple-small")

4 frames
/usr/local/lib/python3.7/dist-packages/transformers/models/gpt2/tokenization_gpt2_fast.py in __init__(self, vocab_file, merges_file, tokenizer_file, unk_token, bos_token, eos_token, add_prefix_space, **kwargs)
    150             pre_tok_class = getattr(pre_tokenizers, pre_tok_state.pop("type"))
    151             pre_tok_state["add_prefix_space"] = add_prefix_space
--> 152             self.backend_tokenizer.pre_tokenizer = pre_tok_class(**pre_tok_state)
    153 
    154         self.add_prefix_space = add_prefix_space

TypeError: PyMetaspace.__new__() got an unexpected keyword argument: str_rep
````

**Comment by LysandreJik** [comment]
Maybe @n1t0 knows what might be happening here.

**Comment by IndianMLGay** [comment]
@n1t0 LysandreJik spoked call you

**Comment by XiaoLiuAI** [comment]
Dose this problem fixed? I also got the same problem, I trained a SentencePieceBPETokenizer, use save() api to persist it as a tokenizer.json file, but got this error while loading. 

I dig into the rust code, and found that Metaspace is defined as a struct with three attributes but the constructor only accept two. I tried to modify the json file and remove the one not in the constructor method, but got another error: 

`Exception: data did not match any variant of untagged enum PyPreTokenizerTypeWrapper at line 1 column 1449
`
Below is the content around char 1449:
`
lse, "normalized": false}], "normalizer": {"type": "NFKC"}, "pre_tokenizer": {"type": "Metaspace", "replacement": "\\u2581", "add_prefix_space": true}, "post_processor": null, "decoder": {"type": "Meta
`
